lib/takeaway.rb

```diff
@@ -19,14 +19,15 @@ def menu
   def place_order(dish, quantity = 1)
     fail "Sorry we don't have that, check our menu" if !@menu.available?(dish)
 
-    @basket[dish.to_sym] += quantity
+    @basket[dish] += quantity
   end
 
   def total
-    @basket
+
```

Comment:
Maybe consider changing your basket to an array - seems easier to iterate over an array than a hash (or it was for me anyway)